README.md

```diff
@@ -1,167 +1,8 @@
 ## ECSWorkshop - EFS Demo
 
-### Purpose
+This repository is part of the ECS Workshop.
 
-The purpose of this repo is to showcase how you can run a stateful container on ECS Fargate. We will be using EFS for the persistent storage layer that the containers will share and use.
+This repository will deploy a VPC to prepare your environment for the [ECS Workshop](https://ecsworkshop.com/).
```

Comment:
This actually is not what this repo does, can you please update?

Reply:
@maishsk thank you for the review. I have updated the text.